provider/resources.go

```diff
@@ -247,6 +247,10 @@ func Provider() tfbridge.ProviderInfo {
 							Description: "The fully qualified image name that was pushed to the registry.",
 							TypeSpec:    schema.TypeSpec{Type: "string"},
 						},
+						"repoDigest": {
+							Description: "The SHA256 digest of the image that was pushed to the registry.",
```

Comment:
This technically does not need to be SHA256, it can be any supported [digest](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests).

Reply:
Ah, good point. What wording would you suggest? Just "The digest of the image that was pushed to the registry"? Should we include your link?

Reply:
How about "The digest of the manifest pushed to the registry, e.g.: `sha256:abcdef...`."

Reply:
Updated!